### ISSUE TITLE
Issues with not updating

### ISSUE THREAD

**Issue by wrac4242** [noisetorch/NoiseTorch#238]
Issue: When clicking the update button on the program, after prompting for my password, it says the update is successful and to restart.
On a restart of the application, it still is prompting to update
before
![image](https://user-images.githubusercontent.com/42539664/162607789-d78c654b-df42-4fcd-88c2-6da1a4a0ab16.png)
after clicking
![image](https://user-images.githubusercontent.com/42539664/162607800-8b6e543e-ac7e-4a83-a56e-f8d9d0c349c2.png)
after restarting
![image](https://user-images.githubusercontent.com/42539664/162607803-8787b324-0a6c-461e-82dd-0e5140b50ce0.png)

version: 0.11.4

log file while updating
[noise.log](https://github.com/lawl/NoiseTorch/files/8458516/noise.log)

log file after restarting it
[noise2.log](https://github.com/lawl/NoiseTorch/files/8458517/noise2.log)

I have tried rebooting my system, which did not fix this issue
system info:
Fedora 34

**Comment by lawl** [comment]
2022/04/10 08:33:14 error extracting tarball into /home/wrac after 2 files, 9 dirs, 1.90622ms: open /home/wrac/.local/bin/noisetorch: text file busy

It's weird, it's complaining that it can't update the noisetorch executable because it's currently running, but that hasn't been a problem until now. The easiest work-around would be to just download the latest release from github and run

    tar -C $HOME -xzf NoiseTorch_x64.tgz

according to the readme again.

There will likely be on overhaul to the update system soon, I don't have a quick fix for this.

**Comment by wrac4242** [comment]
Thanks, should I gather any more information before I install over it?

**Comment by lawl** [comment]
No, I got all the information I need, thanks! It's just that I really don't have a real solution for you right now.

**Comment by Kastakin** [comment]
Same issue here, I'm on Arch Linux

**Comment by AXDOOMER** [comment]
I'll close this as resolved because of the update changes that we've made and the reproducibility. 

Note that the last release of NoiseTorch before the new audited 0.12 release won't be able to upgrade to 0.12 because the update scheme has changed and the old update server no longer exists. It must be downloaded directly from the releases pages. 

Feel free to open a new issue if the next update doesn't work.

**Comment by TheDukeofErl** [comment]
This is definitely an issue and I see it 100% of the time while testing the updater. I'm going to reopen this one... sadly it seems like people might have to do another manual update, but at least they'll see that an update is available...
